src/Command/Input/OperatingSystemOption.php

```diff
@@ -82,6 +82,7 @@ public static function value(InputInterface $input)
             );
         }
 
+        /** @psalm-var string $value */
```

Comment:
This should not be necessary due to the `is_string` check on line 71... are you getting a psalm error here?

Reply:
psalm is drunked on my environment